rust/yarp-sys/tests/parser_tests.rs

```diff
@@ -0,0 +1,217 @@
+// Because this crate is effectively generated code, our testing really just needs to check that the
+// codegen process didn't do something wonky (which would probably be a bug in the `bindgen` crate).
+// As such, we just need to verify that we can use each of the types and functions we care
+// about--not necessarily anything about their functionality (since that's all part of C-yarp).
+
```

Comment:
I'd added this before adding other test files; it's more or less in the README now, so...

- [ ] Remove this comment